packages/rke2-coredns/generated-changes/patch/values.yaml.patch

```diff
@@ -1,17 +1,26 @@
 --- charts-original/values.yaml
 +++ charts/values.yaml
-@@ -3,8 +3,8 @@
+@@ -3,9 +3,9 @@
  # Declare variables to be passed into your templates.
  
  image:
 -  repository: coredns/coredns
--  tag: "1.9.3"
 +  repository: rancher/hardened-coredns
+   # Overrides the image tag whose default is the chart appVersion.
+-  tag: ""
 +  tag: "v1.10.1-build20230406"
    pullPolicy: IfNotPresent
    ## Optionally specify an array of imagePullSecrets.
    ## Secrets must be manually created in the namespace.
-@@ -76,10 +76,10 @@
+@@ -54,7 +54,6 @@
+ 
+ service:
+ # clusterIP: ""
+-# clusterIPs: []
```

Comment:
It's weird that we remove clusterIPs in the values. It's a comment though, so no harm

Reply:
The chart update added support for ClusterIPs as a multi-value replacement for ClusterIP, but we don't support it because we set ClusterIP from the cluster dns address value.